kili/cli/project/member/add.py

```diff
@@ -0,0 +1,141 @@
+"""CLI's project member add subcommand"""
+
+import csv
+import os
+import re
+from typing import Dict, List, Optional, cast
+import warnings
+import click
+
+from kili.client import Kili
+from kili.cli.common_args import Options
+
+
+REGEX_EMAIL = re.compile(
+    r'([A-Za-z0-9]+[.-_])*[A-Za-z0-9]+@[A-Za-z0-9-]+(\.[A-Z|a-z]{2,})+')
+ROLES = ["ADMIN", "TEAM_MANAGER", "REVIEWER", "LABELER"]
+
+
+def read_csv_user(csv_path: str, emails: List[str], roles: List[str]):
+    """read a csv with a user list and update emails and roles """
+    with open(csv_path, 'r', encoding="utf-8") as file:
+        csvreader = csv.reader(file)
+        for row in csvreader:
+            if re.search(REGEX_EMAIL, row[0]):
```

Comment:
it crashes when there is an invalid email without a specific message, is it the expected behavior?

Reply:
It does not crash on my side. Can you send me the csv file you used?